apps/aecore/src/aec_coinbase_tx.erl

```diff
@@ -86,5 +87,10 @@ deserialize([#{<<"type">> := ?CB_TX_TYPE},
 type() ->
     ?CB_TX_TYPE.
 
+for_client(#coinbase_tx{account = Account}) ->
+    #{<<"account">> => base64:encode(Account),
+      <<"type">> => <<"CoinbaseTxObject">>, % swagger schema name
```

Comment:
Any way this can be programmatically checked (at each invocation) or handled with macro/define?
I am concerned that Swagger schema and files may get out of sync.

Reply:
Maybe it is caught be tests though.

Reply:
I didn't like having the magical strings in the codebase either. I couldn't think of a way to bind them with the .yaml file

Reply:
Also swagger doesn't seem to be doing great job validating it (note `"type":"CoinbaseTxObject1"`)
```
$ curl http://localhost:3113/v1/block/pending?tx_objects=true
{"data_schema":"BlockWithTxs","height":1,"nonce":0,"pow":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"prev_hash":"pi7dujKTxyan9xInx4HGW6CZlJobAiU5PiJYO4OllC4=","state_hash":"61PwZy1enBJQGaishVcNxiOF48Glrh6OWtZyMryxp3Y=","target":553713663,"time":1515163659773,"transactions":[{"tx":{"signatures":["MEQCIHSJcY4A6dnQ4XKerLQzw3v84u9MspWo+onMy8Oa/cIWAiACLdMzz32Fuv5yrQ8+E0FWe0PVtxaNZgAoHHeErpI31g=="],"tx":{"account":"BFzDaaQuZ3XnmLs4hNL7q97YgV283vzOQKtvXF0EG5vRiBY4wmDOQN9+hfsKZvpOxUUoZYLEP8LcX7PKpkynV5c=","type":"CoinbaseTxObject1","vsn":1}}}],"txs_hash":"SjYAltlIxPBiCe+QLKm+Ea7DyMrcvx0eUYat4UAbjtQ=","version":3}```